src/client/java/dev/creesch/WebchatClient.java

```diff
@@ -176,14 +182,16 @@ private void showWebAddress(MinecraftClient client) {
         String webchatPort = String.valueOf(
             ModConfig.HANDLER.instance().httpPortNumber
         );
+        String bindAddress = ModConfig.HANDLER.instance().httpBindAddress;
+
         Text message = Text.literal("Web chat: ").append(
-            Text.literal("http://localhost:" + webchatPort)
+            Text.literal("http://" + bindAddress + ":" + webchatPort)
```

Comment:
This needs a different approach. When changing the setting to `0.0.0.0`  it will output `http://0.0.0.0:8080` but that obviously does not work. 
I am not sure if there is a smart approach here, I think we just want this to keep saying localhost, people who change the setting should also be able to figure out the lan IP of their computer.

Reply:
Going to `http://0.0.0.0:8080` works just fine for me on MacOS. But maybe it doesn't on other operating systems?

Reply:
I'll double check tomorrow, but it didn't appear to be working on windows.

Reply:
Just tested it again, it does not work on windows.

Reply:
I am not sure why you marked this as resolved. I said it does **not** work on windows.

Reply:
My bad. I don't recall what happened.

Reply:
I don't think this one is necessary for doing a new release. I'll come back to this when I have time.

Reply:
Alright, I was about to ask if you wanted this in the next release as I did want to start preparing one. We can also do smaller releases after this one as I suspect most of them will be mostly QoL features like this anyway.